bigmac.sh

```diff
@@ -24,9 +25,30 @@ macpro=$(sysctl hw.model)
 if [[ "$macpro" != *"MacPro"* ]]
   then
     echo
-    echo "Please run Big Mac on a Mac Pro."
+    echo "Please run BigMac on a Mac Pro; otherwise please use"
+    echo "Barrykn's Big-Sur-Micropatcher:"
+    echo "    https://github.com/barrykn/big-sur-micropatcher"
```

Comment:
There's tons of different patchers. I have one, ASentientHedgehog has one, and then there's also the micropatcher. Barrykn's is the original, but you shouldn't really suggestion one patcher.